### ISSUE TITLE
Keyword arg for `max_gas_price`

### ISSUE THREAD

**Issue by jackwardell** [fireblocks/fireblocks-sdk-py#63]
**Is your feature request related to a problem? Please describe.**
Sorry me again. This is a tiny issue but I am a pedantic person.

In the method:
```
 def set_gas_station_configuration(self, gas_threshold, gas_cap, max_gas_price):
        """Set configuration of the Gas Station account

        Args:
            gasThreshold (str)
            gasCap (str)
            maxGasPrice (str, optional)
        """

        url = f"/v1/gas_station/configuration"

        body = {
            "gasThreshold": gas_threshold,
            "gasCap": gas_cap,
            "maxGasPrice": max_gas_price
        }

        return self._put_request(url, body)
```

It would be good if `max_gas_price` is optional, for it to be a keyword arg equal to `None`

**Describe the solution you'd like**
`def set_gas_station_configuration(self, gas_threshold, gas_cap, max_gas_price=None):`

**Additional context**
1. Do you accept clients to open PRs on your code? I would have been happy to add this in myself
2. Might be good to make all params optional so people can just change one element of the config

**Comment by jackwardell** [comment]
Update: some buggy behaviour with the API

I called:
```
>>> fireblocks.set_gas_station_configuration(gas_threshold='0.0004', gas_cap='0.0005', max_gas_price=None)
Traceback (most recent call last):
...
raise FireblocksApiException("Got an error from fireblocks server: " + response.text, error_code)
    fireblocks_sdk.api_types.FireblocksApiException: Got an error from fireblocks server: {"message":"Invalid request. maxGasPrice: should be string","code":900}

>>> fireblocks.set_gas_station_configuration(gas_threshold='0.0004', gas_cap='0.0005', max_gas_price='None')
{'success': True}

>>> fireblocks.get_gas_station_info()
{'balance': {'ETH': '0',
  'ETH_TEST': '0',
  'MATIC_POLYGON': '0',
  'BNB_TEST': '0',
  'BNB_BSC': '1',
  'ETH_TEST2': '0',
  'MATIC_POLYGON_MUMBAI': '0'},
 'configuration': {'gasThreshold': '0.0004',
  'gasCap': '0.0005',
  'maxGasPrice': 'None'}}

# now I'm thinking this is odd?

>>> fireblocks.set_gas_station_configuration(gas_threshold='0.0004', gas_cap='0.0005', max_gas_price='hello')
{'success': True}

>>> fireblocks.get_gas_station_info()
{'balance': {'ETH': '0',
  'ETH_TEST': '0',
  'MATIC_POLYGON': '0',
  'BNB_TEST': '0',
  'BNB_BSC': '1',
  'ETH_TEST2': '0',
  'MATIC_POLYGON_MUMBAI': '0'},
 'configuration': {'gasThreshold': '0.0004',
  'gasCap': '0.0005',
  'maxGasPrice': 'hello'}}

```

NB: when I called `fireblocks.get_gas_station_info()` before setting the config:
```
>>> fireblocks.get_gas_station_info()
{'balance': {'ETH': '0',
  'ETH_TEST': '0',
  'MATIC_POLYGON': '0',
  'BNB_TEST': '0',
  'BNB_BSC': '1',
  'ETH_TEST2': '0',
  'MATIC_POLYGON_MUMBAI': '0'},
 'configuration': {'gasThreshold': '0.005', 'gasCap': '0.01'}}
```


So... It seems there is no option to not set `max_gas_price`, moreover when setting it will accept any string, there is no validation that the string is a valid number

**Comment by jackwardell** [comment]
@tomervil @michael-fireblocks sorry to @ you both but I was wondering RE the above if there is any suggested fix? 

I am happy not to have `gasCap` but I don't know how to unset it?

**Comment by yarinvak** [comment]
Hey @jackwardell we released version 1.4.0 that should resolve your issue, also the bug in our API was fixed.

Thanks!

**Comment by jackwardell** [comment]
thank you :)
